configure

```diff
@@ -223,6 +224,11 @@ function install_pkgin
 		    "failed to install $pkg"
 		fi
 	done
+
+	#
+	# Make sure we've upgraded all of our installed packages.
+	#
+	$conf_priv pkgin -y upgrade || fatal "failed installed-package upgrades"
```

Comment:
@jperkin suggested I move this before any installations. I'm likely to move it to the beginning of this function.  Please confirm?

Reply:
Nope, I'm wrong: move it to after `update` but before installations.